setup almost done

kschmott opened the pull request.

kschmott committed:
setup almost done

santiagomed committed:
gitignore fix

santiagomed committed:
fixed tsc build errors; added build scripts

santiagomed committed:
testing lib fix

santiagomed committed:
added testing stuff; fixed some html

santiagomed reviewed (approved):
I just added scripts for building client and server separately, added some testing setup stuff and redundant HTML. Feel free to go over the small changes I made. For example, you guys were using labels redundantly; and had a div for the close button instead of a button.

kschmott commented:
> I just added scripts for building client and server separately, added some testing setup stuff and redundant HTML. Feel free to go over the small changes I made. For example, you guys were using labels redundantly; and had a div for the close button instead of a button.

The reason we were using labels was because that's how they used them in the daisy ui docs. I'm not sure it if matters or not, but it seems to work as buttons.

santiagomed commented:
> The reason we were using labels was because that's how they used them in the daisy ui docs. I'm not sure it if matters or not, but it seems to work as buttons.

Mmm that's odd, it still works with the changes I made but can you pass the link to the docs for me to double check? Labels are usually paired with input tags to describe them. The `forHtml` attribute points to the id in the input tag. Check [this](https://www.w3schools.com/tags/tag_label.asp) out as an example.

kschmott committed:
added navbar skeleton and routing structure

santiagomed committed:
tests

kschmott committed:
Merge pull request #10 from void-tech-um/navbar

Navbar
